src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetWebApi2Integration.cs

```diff
@@ -139,21 +140,23 @@ public static class AspNetWebApi2Integration
         /// <typeparam name="T">The type of the generic Task instantiation</typeparam>
         /// <param name="instrumentedMethod">The underlying ExecuteAsync method</param>
         /// <param name="apiController">The Api Controller</param>
-        /// <param name="controllerContext">The controller context for the call</param>
+        /// <param name="context">The controller context for the call</param>
         /// <param name="cancellationToken">The cancellation token</param>
         /// <returns>A task with the result</returns>
         private static async Task<T> ExecuteAsyncInternal<T>(
             Func<object, object, CancellationToken, object> instrumentedMethod,
             object apiController,
-            object controllerContext,
+            object context,
             CancellationToken cancellationToken)
         {
+            var controllerContext = context.As<IHttpControllerContext>();
```

Comment:
We should revisit this when we merge this: https://github.com/DataDog/dd-trace-dotnet/pull/1220

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1208/commits/a13a1df56f76b3e06c54a653676e8c68dcc4fa66